Disable failing tests in Python 3.7

My current work-around disabling some file-based tests in python 3.7 seems to have missed a couple:

```
=========================== short test summary info ============================
FAILED test/test_extractor.py::TestExtractFileDeb::test_extract_file_deb - Ru...
FAILED test/test_helper_script.py::TestHelperScript::test_scan_files_version
===== 2 failed, 360 passed, 341 skipped, 114 warnings in 88.14s (0:01:28) ======
```

I would still like to know *why* those are failing only sometimes on 3.7 so we could make it so that they don't fail, but in the meantime I think the best solution is to disable them on 3.7 only.  They're running just fine on 3.8, 3.9 and 3.10 so I think we have sufficient coverage.

I'm going to mark this as a good first issue.  You can follow the example of what I did here to disable these two tests:
https://github.com/intel/cve-bin-tool/commit/376d996fa97917e3049cdabadb28a2a4294d9c2b

Basically, find the two tests named above and add pytest skipif line above them:
```
@pytest.mark.skipif(sys.version_info < (3, 8), reason="py3.7 fails sometimes")
```

I suspect this should be two one-line fixes, one in test/test_extractor.py and one in test/test_helper_script.py

--- a/test/test_extractor.py
+++ b/test/test_extractor.py
@@ -251,6 +251,7 @@ class TestExtractFileDeb(TestExtractorBase):
         return self.extractor.file_extractors[self.extractor.extract_file_deb]
 
     @pytest.mark.asyncio
+    @pytest.mark.skipif(sys.version_info < (3, 8), reason="py3.7 fails sometimes")
     async def test_extract_file_deb(self, extension_list: List[str]):
         """Test the deb file extraction"""
         async for extracted_path in self.extract_files(

--- a/test/test_helper_script.py
+++ b/test/test_helper_script.py
@@ -69,6 +69,7 @@ class TestHelperScript:
             scan_files(args)
             assert "PRODUCT_NAME not in arguments" in caplog.text
 
+    @pytest.mark.skipif(sys.version_info < (3, 8), reason="py3.7 fails sometimes")
     def test_scan_files_version(self, caplog):
         args = {
             "filenames": [

